modules/vsphere-problem-detector-viewing-logs.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/using-vsphere-problem-detector-operator.adoc
+
+:operator-name: vSphere Problem Detector Operator
+
+[id="vsphere-problem-detector-viewing-logs_{context}"]
+= Viewing the logs from the vSphere Problem Detector Operator
+
+After the {operator-name} runs and performs the configuration checks, it creates log records that can be viewed from the command line or from the {product-title} web console.
+
+.Procedure
+
+* To view the logs by using the command line, run the following command.
++
+[source,terminal]
+----
+$ oc logs deployment/vsphere-problem-detector-operator \
+    -n openshift-cluster-storage-operator
+----
+
+* To view the Operator logs with the {product-title} web console.
+
+. Navigate to *Workloads* -> *Pods*.
```

Comment:
I suggest nesting the substeps under the introductory step by s/./.. for each list item.